### PR TITLE
Develop script for `Incorrectly parsed HTML elements` and `Missing type annotations` Issues

### PR DESCRIPTION
**Pull request by v-weiyding**
1. 
**Testcase:**
Missing type annotations.  - Develop script `TestPageAnnotation.cs` and `MissingTypeAnnotation.cs`.

**Design:**
- Create a browser instance.
- Get all the class and method parameters in the test page.
- Check each class and method parameter for correct type annotations and record any missing or incorrect ones.
    - Extract parameter maps for classes and methods. 
    - If a parameter is found to be missing a type annotation, add it to the `errorList`.
    - If the argument list for the class is empty, add it to the `errorList`.
- Log the error message in the report, need engineer's manual check.

**Result:**
[link](https://learn.microsoft.com/en-us/python/api/azure-batch/azure.batch.operations.accountoperations?view=azure-python)
image

2. 
**Testcase:**
Incorrectly parsed HTML elements. - Develop script `TestPageLabel.cs` and `ExtraLabelValidation.cs`.

**Design:**
- Create browser instance.
- Define a list `(labelList)` containing various HTML tags and entities.
- Iterate through `labelList` and check if the page content contains any of the tags. If any tags are found, add them to the `errorList`.
- Log the error message in the report, need engineer's manual check.

**Result:**
[link](https://learn.microsoft.com/en-us/python/api/azure-security-attestation/azure.security.attestation.aio.attestationadministrationclient?view=azure-python)
image

@zedy-wj and @v-xuto for notification.

### CODE REVIEW

File: ContentValidation/ExtraLabelValidation.cs [hunk 1]
```diff
@@ -41,12 +44,17 @@ public async Task<TResult> Validate(string testLink)
...
         var browser = await _playwright.Chromium.LaunchAsync(new BrowserTypeLaunchOptions { Headless = true });
         var page = await browser.NewPageAsync();
         await page.GotoAsync(testLink);
 
         var text = await page.Locator("html").InnerTextAsync();
 
+
+        // Iterate through labelList and check if the page content contains any of the tags. If any tags are found, add them to the errorList.
+        int sum = 0;
+        string ErrorInfo = "Extra label found: ";
```
Comment (zedy-wj) [inline ContentValidation/ExtraLabelValidation.cs:57]: Update to: errorInfo.

Comment (v-weiyding) [reply ContentValidation/ExtraLabelValidation.cs]: @zedy-wj , I have updated by your comments, please re-review.

**Review by zedy-wj — APPROVED** [review]

**Comment by zedy-wj** [comment]
@v-xuto - Design has not changed. Please review it, thanks.

File: ContentValidation/MissingTypeAnnotation.cs [hunk 2]
```diff
@@ -12,26 +12,39 @@ public MissingTypeAnnotation(IPlaywright playwright)
...
         _playwright = playwright;
     }
-    public async Task<TResult> Validate(string testLink)
+    public async Task<TResultNew> Validate(string testLink)
     {
-        var res = new TResult();
+        var res = new TResultNew();
+        List<string> errorList = new List<string>();
 
         var browser = await _playwright.Chromium.LaunchAsync(new BrowserTypeLaunchOptions { Headless = true });
```
Comment (v-xuto) [inline ContentValidation/MissingTypeAnnotation.cs:20]: Add comments: Create a browser instance.

Comment (v-weiyding) [reply ContentValidation/MissingTypeAnnotation.cs:20]: @v-xuto, I have updated by your comments, please re-review.

**Comment by zedy-wj** [comment]
@v-weiyding - Please update the file name `MissingTypeAnnotation.cs` to `TypeAnnotationValidation.cs`.

**Comment by v-xuto** [comment]
@zedy-wj LGTM, please merge.

**Pull request merged**